java/libs.javacapi/external/binaries-list

```diff
@@ -14,4 +14,4 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-267AFC87A08301145B10D86DA674724B884F010C nb-javac-15-api.jar
\ No newline at end of file
+A152261F8D708A5E1A7FCDAA3981663E nb-javac-15.0.0.1-api.jar
```

Comment:
It would be better if @eppleton can upload these JARs to Maven central. Then we can use Maven co-ordinates and SHA1 content just as we do with [HTML/Java JARs](https://github.com/apache/netbeans/blob/master/platform/net.java.html.boot.fx/external/binaries-list#L17).